commit f7f2317beecf6cb4ef49a76f602468eee26c8a6a
Author: Stefano Gurrieri <stefanogurrieri@roj.com>
Date:   Fri May 15 08:26:48 2015 +0200

    Kernel command line changed to manage lvds and hdmi displays.

diff --git a/include/configs/mx6roj_common.h b/include/configs/mx6roj_common.h
index 4184b4e..3de2896 100644
--- a/include/configs/mx6roj_common.h
+++ b/include/configs/mx6roj_common.h
@@ -159,7 +159,7 @@
 			"fi; "	\
 		"fi\0" \
 	EMMC_ENV	  \
- "mmcsetroot=setenv mmcroot /dev/mmcblk${mmcdev}p${mmcrootpart} rootwait rw\0" \
+ "mmcsetroot=setenv mmcroot /dev/mmcblk${mmcdev}p${mmcrootpart} rootwait rw video=mxcfb0:dev=hdmi,1920x1080M@60,if=RGB24,fbpix=RGB32 video=mxcfb1:dev=ldb,800x600M@60,if=RGB24,fbpix=RGB32 ldb=sin0\0" \
 	"mmcargs=run mmcsetroot;setenv bootargs console=${console},${baudrate} " \
 		"root=${mmcroot}\0" \
 	"loadbootscript=" \
